### ISSUE TITLE
Restify test fails on Node 4

### ISSUE THREAD

**Issue by micahr** [restify/node-restify#916]
https://travis-ci.org/restify/node-restify/jobs/79522234

```
✖ GH-149 limit request body size
AssertionError: 500 == 413
    at Object.equal (/home/travis/build/restify/node-restify/node_modules/nodeunit/lib/types.js:83:39)
    at ClientRequest.<anonymous> (/home/travis/build/restify/node-restify/test/server.test.js:804:11)
    at ClientRequest.g (events.js:260:16)
    at emitOne (events.js:77:13)
    at ClientRequest.emit (events.js:169:7)
    at HTTPParser.parserOnIncomingClient (_http_client.js:415:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:88:23)
    at Socket.socketOnData (_http_client.js:305:20)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)

✖ GH-149 limit request body size (json)
AssertionError: 500 == 413
    at Object.equal (/home/travis/build/restify/node-restify/node_modules/nodeunit/lib/types.js:83:39)
    at ClientRequest.<anonymous> (/home/travis/build/restify/node-restify/test/server.test.js:834:11)
    at ClientRequest.g (events.js:260:16)
    at emitOne (events.js:77:13)
    at ClientRequest.emit (events.js:169:7)
    at HTTPParser.parserOnIncomingClient (_http_client.js:415:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:88:23)
    at Socket.socketOnData (_http_client.js:305:20)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
```

**Comment by micahr** [comment]
It looks like Node changed the Http Status codes in 4: https://github.com/nodejs/node/commit/2d9456e3e61cf969d5cd11ad85484aa961844ff6#diff-feaf3339998a19f0baf3f82414762c22R53

Specifically related to these tests, there is no longer a RequestEntityTooLargeError, instead it is PayloadTooLarge now.

**Comment by micahr** [comment]
Here is the PR on the Node side: https://github.com/nodejs/node/pull/1470

**Comment by fszakka** [comment]
just used restify with node 4 and retrieved from mssql over 4000 records no problem.

**Comment by micahr** [comment]
The problem I'm addressing is that our test suite fails on Node 4.

**Comment by micahr** [comment]
Fixed by #917
